dll/win32/syssetup/syssetup.rc

```diff
@@ -34,16 +34,16 @@ IDI_ICON5 ICON "resources/5.ico"
 STRINGTABLE
 BEGIN
     IDS_ACKPROJECTS "\
-Wine - http://www.winehq.org\
-\nFreeType - http://www.freetype.org\
+Wine - http://winehq.org\
+\nFreeType - http://freetype.org\
```

Comment:
tbh. this commit isn't useful; having the `www.` removed is more unnatural than anything else.

Reply:
I wouldn't see why this would be unnatural at all. Some links also had no www placed there before, and you never complained the last 10 years about that feeling unnatural. Vast majority of browsers (above IE6) do not even show that anymore. To me this feels like 1990.
Furthermore, since the links cannot be clicked, but are a pen&paper-exercise for anyone willing to follow them, I do perceive it as an upgrade in useability even if they have not to write down that much on their paper.

Reply:
If a website works with or without the www part is up to the server the website is running from, it is not in any way a thing that can be or should be stripped or determined client side

Reply:
What is offered is fine to be used.